test/create.test.js

```diff
@@ -1,7 +1,7 @@
-const
-  should = require('should'),
-  PluginLocal = require('../lib'),
-  PluginContext = require('./mock/pluginContext.mock.js');
+const should = require('should');
+const PluginLocal = require('../lib');
+const PluginContext = require('./mock/pluginContext.mock.js');
+const { KuzzleRequest } = require('kuzzle');
```

Comment:
Nitpicking: it should be global module first
```suggestion
const should = require('should');
const { KuzzleRequest } = require('kuzzle');

const PluginLocal = require('../lib');
const PluginContext = require('./mock/pluginContext.mock.js');
```